resources/home/dnanexus/run_workflows/tests/test_manage_dict.py

```
@@ -598,88 +599,94 @@
-    def test_adding_sample_prefix(self):
+    def test_adding_sample_prefix(self, other_inputs):
         """
         Test for finding INPUT-SAMPLE-PREFIX and replacing with sample prefix
         """
         assert (
-            self.output["sample_name_prefix"] == "my_sample"
+            other_inputs["sample_name_prefix"] == "my_sample"
         ), "INPUT-SAMPLE-PREFIX not correctly replaced"
 
-    def test_adding_project_id(self):
+    def test_adding_project_id(self, other_inputs):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_adding_project_id`

Add return type annotation: `None`

(ANN201)